nextflow.config

```diff
@@ -252,45 +259,153 @@ docker.registry      = 'quay.io'
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/crisprseq'
-    author          = """Júlia Mir Pedrol, Laurence Kuhlburger"""
+    author          = """Júlia Mir Pedrol, Laurence Kuhlburger""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'Júlia Mir Pedrol',
+            affiliation: 'Centre for Genomic Regulation (CRG)',
+            email: 'mirp.julia@gmail.com',
+            github: 'https://github.com/mirpedrol',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: 'https://orcid.org/0000-0001-6104-9260'
+        ],
+        [
+            name: ' Laurence Kuhlburger',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/LaurenceKuhl',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Matthias Hörtenhuber',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/mashehu',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Alan Tracey',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/alan-tracey',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Jasmin Frangenberg',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/jasmezz',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Marta Sanvicente-García',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/msanvicente',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
```

Comment:
Hi @msanvicente,
we have added all the contributors to the pipeline and you contributed to crisprseq. Could you please update your data here? 
Thanks!

(I wasn't sure if your user is this one or @MartaSanvicente )